suite-native/video-assets/scripts/compress-videos.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+# Reads all .mp4 files from `assets/raw` and creates a compressed copy in `assets/compressed` if not already existing.
+
+cd assets/raw || exit
+for video_file in *.mp4
+do
+    if [ ! -e "../compressed/$video_file" ]
```

Comment:
I would remove this condition and just always regenerate files. It's very useful if you tweak settings or something.

Reply:
fixed 781b757df